appveyor.yml

```diff
@@ -126,8 +126,7 @@ before_deploy:
   - "SET PATH=%ORIGPATH%"
   - "%CMD_IN_ENV% %WP_INSTDIR%/scripts/env.bat"
   - "%CMD_IN_ENV% python.exe setup.py install"
-  - "%CMD_IN_ENV% easy_install --upgrade pip"
-  - "%CMD_IN_ENV% pip install --upgrade configobj traitsui"
+  - "%CMD_IN_ENV% pip install --upgrade configobj traitsui start_jupyter_cm"
```

Comment:
@vidartf, although I can't reproduce the issue, in prevention I've added a line to patch all shebangs to appveyor. Is this ready to merge then?